CHANGELOG.md

```diff
@@ -19,9 +19,129 @@ const expandWidget = new Expand({
 })
 ```
 
+### View Popup deprecations and changes
+
+In a continuous effort to optimize the performance and load time of the API, the Popup widget will no longer be bundled with the MapView and SceneView. The Popup widget receives new features regularly, such as when browsing related records. Each new feature added to the Popup widget increases the size of the widget and amount of built code that is sent to the web browser. For example, the Popup module, which is bundled with the MapView, represents around 50% of the size of the bundle.
```

Comment:
**question**: Is this an SDK or an API after the name change?

```suggestion
In a continuous effort to optimize the performance and load time of the SDK, the Popup widget will no longer be bundled with the MapView and SceneView. The Popup widget receives new features regularly, such as when browsing related records. Each new feature added to the Popup widget increases the size of the widget and amount of built code that is sent to the web browser. For example, the Popup module, which is bundled with the MapView, represents around 50% of the size of the bundle.
```

Reply:
We are still adjusting! :D

Reply:
Right, SDK's contain API's. Makes sense! ✌️